libcipher/caesar.py

```diff
@@ -0,0 +1,64 @@
+__author__ = 'kskrishnasangeeth'
+
+import random
+import sys
+
+
+def encrypt(message, key=None):
+    """Returns the encrypted messsage for the string passed as argument.
+
+    Uses a random key for generating the cipher text
```

Comment:
Selecting a random key, that user itself will not know, does not seem to be useful. I don't think this is required.